bitnami/dataplatform-bp1/values.yaml

```diff
@@ -63,9 +63,9 @@ dataplatform:
     ## @param dataplatform.exporter.image.pullSecrets Specify docker-registry secret names as an array
     ##
     image:
-      registry: docker.io
-      repository: bitnami/dataplatform-exporter
-      tag: 0.0.11-scratch-r4
+      registry: harbor-repo.vmware.com
+      repository: octo_data_platforms/dataplatform-exporter
+      tag: 1.0.0
```

Comment:
Could you please revert these lines to the previous values? The tag will be automatically updated by our CI system before emerging the PR.